{
    "type": "bundle",
    "id": "bundle--c0467f0d-f7d9-4875-90db-5e1aa1fd65ab",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--991a121d-c3fd-4e42-9405-aa61c3ce1b6e",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:12:00.58166Z",
            "modified": "2021-03-13T20:12:00.58166Z",
            "name": "CVE-2019-16872",
            "description": "Portainer before 1.22.1 has Incorrect Access Control (issue 1 of 4).",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2019-16872"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}